src/sorting.cpp

```diff
@@ -133,3 +133,45 @@ std::vector<int> counting_sort(std::vector<int>& vec, int k) {
 
   return result;
 }
+
```

Comment:
I decided to skip the description as we have started adding info to Readme.md. Maybe we should still have descriptions in the above the functions? @holmgr @fumoffu947 @Adzox